packages/components-js/scripts/convertToNextJsVRTPage.ts

```diff
@@ -1,20 +1,32 @@
 import * as fs from 'fs';
 import * as path from 'path';
-import { paramCase } from 'change-case';
 import { convertToReactVRTPage, ReactCharacteristics } from './convertToReactVRTPage';
+import { paramCase } from 'change-case';
 
 const sourceBasePath = path.resolve(__dirname, '../../components-react/src');
 const pollComponentsReadyFilePath = path.resolve(sourceBasePath, 'pollComponentsReady.ts');
-const pollComponentsReadyFileContent = fs.readFileSync(pollComponentsReadyFilePath, 'utf8');
+const pollComponentsReadyFileContent = fs
+  .readFileSync(pollComponentsReadyFilePath, 'utf8')
+  .replace(/export (const pollComponentsReady)/, '$1');
```

Comment:
Perhaps this shouldn't happen here when the variable name is called `pollComponentsReadyFileContent` and now the content was manipulated?